MANUAL.md

```diff
@@ -128,80 +136,69 @@ plugins:
       - import_extension=js
 ```
 
-With [`protoc`][gh-protoc], multiple options are specified with multiple `--es_opt` flags. Alternatively, both compilers
-allow to specify multiple options as a single comma-separated value like `target=ts,import_extension=js`.
+With [`protoc`][gh-protoc], you specify multiple options with multiple `--es_opt` flags. Alternatively, both compilers
+allow you to specify multiple options as a single comma-separated value like `target=ts,import_extension=js`.
 
-> **TODO** include all options from https://github.com/bufbuild/protobuf-es/tree/v2/packages/protoc-gen-es#plugin-options here.
+### `target`
 
-## Generated code
+This option controls whether the plugin generates JavaScript, TypeScript, or TypeScript declaration files. Possible 
+values:
 
-This section shows what code is generated for any given Protobuf definition. For a quick overview, here is the
-TypeScript declaration we generate for [example.proto]:
+- `target=js`: Generates a `_pb.js` file for every `.proto` input file.
+- `target=ts`: Generates a `_pb.ts` file for every `.proto` input file.
+- `target=dts`: Generates a `_pb.d.ts` file for every `.proto` input file.
 
-```typescript
-// @generated by protoc-gen-es v2.0.0 with parameter "target=dts"
-// @generated from file example.proto (package example, syntax proto3)
-/* eslint-disable */
+You can pass multiple values by separating them with `+`—for example, `target=js+dts`.
 
-import type { GenFile, GenMessage } from "@bufbuild/protobuf/codegenv1";
-import type { Message } from "@bufbuild/protobuf";
+By default, it generates JavaScript and TypeScript declaration files, which produces the smallest code size and is the 
+most compatible with various bundler configurations. If you prefer to generate TypeScript, use `target=ts`.
 
-/**
- * Describes the file example.proto.
- */
-export declare const file_example: GenFile;
+### `import_extension`
 
-/**
- * @generated from message example.User
- */
-export declare type User = Message<"example.User"> & {
-  /**
-   * @generated from field: string first_name = 1;
-   */
-  firstName: string;
+By default, [protoc-gen-es] doesn't add file extensions to import paths. However, some environments require an import 
```

Comment:
We don't have a link reference for `[protoc-gen-es]`. Should we change this to `[@bufbuild/protoc-gen-es]`?

It's the npm package name for the same thing.